src/Controls/src/Core/HandlerImpl/Label/Label.Windows.cs

```diff
@@ -1,15 +1,40 @@
 #nullable disable
+using System.Collections.Generic;
 using Microsoft.Maui.Controls.Platform;
+using Microsoft.UI.Xaml.Controls;
 
 namespace Microsoft.Maui.Controls
 {
 	public partial class Label
 	{
+		TextBlock _textBlock;
+
+		private protected override void OnHandlerChangedCore()
+		{
+			base.OnHandlerChangedCore();
+
+			if (Handler is not null)
+			{
+				if (Handler is LabelHandler labelHandler && labelHandler.PlatformView is TextBlock textBlock)
+				{
+					_textBlock = textBlock;
```

Comment:
Why do we need the native view change here? can't we look at size changes on the Label itself ? Override Measure or something ?